fission/src/ui/components/DragModeIndicator.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react"
+import Label, { LabelSize } from "./Label"
+import { FaHandPaper } from "react-icons/fa"
+
+export default function DragModeIndicator() {
+    const [enabled, setEnabled] = useState<boolean>(false)
+
+    useEffect(() => {
+        const handleDragModeToggle = (event: CustomEvent) => {
+            setEnabled(event.detail.enabled)
+        }
+
+        window.addEventListener("dragModeToggled", handleDragModeToggle as EventListener)
+
+        return () => {
+            window.removeEventListener("dragModeToggled", handleDragModeToggle as EventListener)
+        }
+    }, [])
+
+    const handleClick = () => {
+        window.dispatchEvent(new CustomEvent("disableDragMode"))
```

Comment:
This doesn't seem to trigger the "disabled" toast, not a big deal but it should probably be consistent with disabling in the menu